bylaws/004-elections-and-vacancies.md

```diff
@@ -2,12 +2,14 @@
 
 ## Election Calendar
 
-Both Secretaries and Core Committee members are elected for two year terms with votes staggered eight months apart.  One election is held in January of even numbered years, one election is held in August of even numbered years, and one election is held in May of odd numbered years.  For example there would be elections in January 2018, August 2018 and May 2019.  Each regular election includes a single Secretary and four (4) Core Committee members, plus any potential unplanned vacancies.
+Both Secretaries and Core Committee members are elected for two year terms with votes staggered eight months apart. One election is held in January of even numbered years, one election is held in August of even numbered years, and one election is held in May of odd numbered years. For example there would be elections in January 2018, August 2018 and May 2019. Each regular election includes a single Secretary and four (4) Core Committee members, plus any potential unplanned vacancies.
 
-The election calendar will be established by the Secretaries at least one week before the start of the election month.  Nominations must run for a minimum of seven (7) days.  The vote will open immediately upon the end of the nomination period and must run for fourteen (14) days.  The new Secretary will take office at 5pm UTC time on the last Sunday of the month, and the voting period must conclude at least 24 hours prior to that.
+The election calendar will be established by the Secretaries at least one week before the start of the election month. Nominations must run for a minimum of seven (7) days. The vote will open immediately upon the end of the nomination period and must run for fourteen (14) days. The new Secretary will take office at 5pm UTC time on the last Sunday of the month, and the voting period must conclude at least 24 hours prior to that.
 
 The election process will be overseen by either of the Secretaries whose term is not expiring, and both the Secretary and Core Committee elections may be overseen by the same Secretary or different Secretaries.
 
+An individual may stand in both the secretary and Core Committee election simultaneously, and should they be elected to both, they may have the choice of which role to accept unless there are no other candidates in the election to fill the position they would vacate.
```

Comment:
This concerns me. Mainly that I think it introduces too much complexity.  Also, why would someone do that?  They're very different jobs.